autocontingency.lic

```diff
@@ -0,0 +1,60 @@
+=begin
+  
+=end
+custom_require.call(%w(common common-arcana events spellmonitor drinfomon))
+class Contingency
+  include DRC
+  include DRCA
+  include DRCT
+
+  def initialize
+    arg_definitions = [
+      [
+        { name: 'anchor', regex: /\w+/, description: 'Contingency to create' },
+        { name: 'room', regex: /\d+/, description: 'Room to move to' }
+      ],
+      [
+        { name: 'invoke', regex: /invoke/i }
+      ]
+    ]
+    args = parse_args(arg_definitions)
+    @settings = get_settings
+    @anchor = args.anchor.capitalize
+    @room = args.room
+    if args.invoke
+      check_contingency
+    else
+      contingency_saferoom
```

Comment:
💡 Minor, but I recommend passing `args.anchor.capitalize` and `args.room` as parameters to `contingency_saferoom` instead of making instance vars